### ISSUE TITLE
class imbalance issue of PHOC representation

### ISSUE THREAD

**Issue by pinakinathc** [ssudholt/phocnet#14]
@ssudholt 
when calculating a 604 PHOC representation on GW handwritten dataset,
I used your function: ```build_phoc_descriptor```
Now, after generating the PHOC representation,
97% of the elements out of 604 are 0's

and 3% of the elements out of 604 are 1's.

As you see, this is a huge class imbalance. Hence, during training, the training and validation loss (used log_loss as mentioned in the paper) decreases (as all elements learns to give value close to 0) but the MAP never increases.

Can you let me know how you figured the solution of this problem?
